.github/ISSUE_TEMPLATE/release.md

```diff
@@ -8,88 +8,98 @@ assignees: ''
 
 # Open Match Release Process
 
-Follow these instructions to create an Open Match release.  The output of the
-release process is new images and new configuration.
+Follow these instructions to create an Open Match release. The output of the
+release process are new images and new configuration.
```

Comment:
This is/are is matching the word "output".  As output is singular, "is" is correct here.  Optionally "output" could be changed to "outputs", in which case "are" would be correct.